src/overlays/actors/ovl_En_Crow/z_en_crow.c

```diff
@@ -88,60 +103,463 @@ static DamageTable D_8099C0AC = {
+    if (this->actor.parent != NULL) {
+        this->actor.flags &= ~ACTOR_FLAG_1;
+    }
+    EnCrow_SetupFlyIdle(this);
+}
+
+void EnCrow_Destroy(Actor* thisx, GlobalContext* globalCtx) {
+    EnCrow* this = THIS;
+
+    Collider_DestroyJntSph(globalCtx, &this->collider);
+}
+
+void EnCrow_SetupFlyIdle(EnCrow* this) {
+    this->timer = 100;
+    this->collider.base.acFlags |= AC_ON;
+    this->actionFunc = EnCrow_FlyIdle;
+    this->skelAnime.playSpeed = 1.0f;
+}
+
+void EnCrow_FlyIdle(EnCrow* this, GlobalContext* globalCtx) {
+    Player* player = GET_PLAYER(globalCtx);
+    f32 sp38;
+    s32 skelanimeUpdated;
+    s16 temp;
+
+    SkelAnime_Update(&this->skelAnime);
+    skelanimeUpdated = Animation_OnFrame(&this->skelAnime, 0.0f);
+    this->actor.speedXZ = (Rand_ZeroOne() * 1.5f) + 3.0f;
+
+    if ((this->actor.parent != NULL) && ((this->actor.parent->home.rot.z) == 0)) {
+        this->actor.home.pos.x = this->actor.parent->world.pos.x;
+        this->actor.home.pos.z = this->actor.parent->world.pos.z;
+        sp38 = Actor_XZDistanceToPoint(&this->actor, &this->actor.parent->world.pos);
+    } else {
+        sp38 = 450.0f;
+        this->actor.flags |= ACTOR_FLAG_1;
+    }
+
+    if (this->actor.bgCheckFlags & 8) {
+        this->aimRotY = this->actor.wallYaw;
+    } else if (Actor_XZDistanceToPoint(&this->actor, &this->actor.home.pos) > 300.0f) {
+        this->aimRotY = Actor_YawToPoint(&this->actor, &this->actor.home.pos);
+    }
+
+    if ((Math_SmoothStepToS(&this->actor.shape.rot.y, this->aimRotY, 5, 0x300, 0x10) == 0) && (skelanimeUpdated) &&
+        (Rand_ZeroOne() < 0.1f)) {
+
+        temp = (Actor_YawToPoint(&this->actor, &this->actor.home.pos) - this->actor.shape.rot.y);
+        if (temp > 0) {
+            this->aimRotY += Rand_S16Offset(0x1000, 0x1000);
+        } else {
+            this->aimRotY -= Rand_S16Offset(0x1000, 0x1000);
+        }
+        Actor_PlaySfxAtPos(&this->actor, NA_SE_EN_KAICHO_CRY);
+    }
+
+    if ((this->actor.depthInWater > -40.0f) || (this->actor.bgCheckFlags & 1)) {
+        this->aimRotX = -0x1000;
+    } else if (this->actor.world.pos.y < (this->actor.home.pos.y - 50.0f)) {
+        this->aimRotX = -Rand_S16Offset(0x800, 0x800);
+    } else if (this->actor.world.pos.y > (this->actor.home.pos.y + 50.0f)) {
+        this->aimRotX = Rand_S16Offset(0x800, 0x800);
+    }
+
+    if ((Math_SmoothStepToS(&this->actor.shape.rot.x, this->aimRotX, 0xA, 0x100, 8) == 0) && skelanimeUpdated &&
+        (Rand_ZeroOne() < 0.1f)) {
+        if (this->actor.home.pos.y < this->actor.world.pos.y) {
+            this->aimRotX -= Rand_S16Offset(0x400, 0x400);
+        } else {
+            this->aimRotX += Rand_S16Offset(0x400, 0x400);
+        }
+
+        this->aimRotX = CLAMP(this->aimRotX, -0x1000, 0x1000);
+    }
+
+    if ((this->actor.bgCheckFlags & 1) != 0) {
+        Math_ScaledStepToS(&this->actor.shape.rot.x, -0x100, 0x400);
+    }
+
+    DECR(this->timer);
+    if ((this->timer == 0) &&
```

Comment:
It does not :/ Would it be better to expand it out instead of using the macro? 
![image](https://user-images.githubusercontent.com/20159000/156905852-616e61ca-1288-40ea-9c34-a34af18558f3.png)